brozzler/__init__.py

```diff
@@ -159,18 +136,18 @@ def behavior_script(url, template_parameters=None, behaviors_dir=None):
                 behavior["behavior_js_template"]
             )
             script = template.render(parameters)
-            logging.info(
-                "using template=%r populated with parameters=%r for %r",
-                behavior["behavior_js_template"],
-                json.dumps(parameters),
-                url,
+            logger.info(
+                "rendering template",
+                template=behavior["behavior_js_template"],
+                parameters=json.dumps(parameters),
```

Comment:
do we need to `dumps` the parameters? Assuming later use of a json log format, deferring serialization to the logger will probably make it easier to query into `parameters`

Reply:
Good catch - we definitely don't need to do that anymore!